engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -114,6 +115,9 @@
     private final ShaderManager shaderManager;
     private final Camera playerCamera;
 
+    // TODO: @In
+    private final OpenVRProvider vrProvider = OpenVRProviderSingleton.vrProvider;
```

Comment:
A pattern I have noticed in Terasology's codebase is to initialize variables not on declaration (which might happen much sooner than instantiation) but on construction or initialization. I'd recommend that also in this case.